### ISSUE TITLE
H1 and H2 headings should only be printed on odd pages (recto side)

### ISSUE THREAD

**Issue by dormerod** [Kashomon/gpub#20]
When reviewing printed drafts, it seems like it would look more normal/professional to only print H1 and H2 headings (Book and Part headings) on the recto side, leaving a blank page on the verso side if necessary.

From memory we collapsed this down earlier to save paper, because all headings were appearing on the recto side, leaving too many blank pages.

I don't think it matters which side H3 and H4 (Chapter and Section headings) appear on. It seems natural for them to appear on either side based on personal experience with books. However, @Kashomon, I also wonder whether the CMoS has something to say about this?

It seems like the current behavior is for only H1 headings to be forced to the recto side.

**Comment by Kashomon** [comment]
personally I think H1-H3 should probably be on the recto + a black page if necessary. But in any case, I'm not sure what the configuration looks like in the memoir class.  Low priority for now.

**Comment by dormerod** [comment]
I agree, possibly H3 as well, depending a bit on how H3 is being used.

**Comment by dormerod** [comment]
For the way we're laying out Relentless, I think your suggestion of putting H1 to H3 on the recto side makes sense (because there's only one chapter per game).

**Comment by dormerod** [comment]
This is a blocking issue for Relentless. To summarize what we've discussed previously; anything above \section (\book, \part and \chapter) should be recto + blank page. Sections should just continue on without new or blank pages, as they do now (otherwise it will mess up the layout throughout the book). TeX seems to handle whether or not to make a new page for a section automatically, depending on how long it is.

I think an earlier version of GPub behaved in exactly the way we want (before the blank pages were removed).
